awscli/examples/rum/batch-get-rum-metric-definitions.rst

```diff
@@ -0,0 +1,39 @@
+**Retrieves the list of metrics and dimensions that a RUM app monitor is sending to a single destination**
+
+The following ``batch-get-rum-metric-definitions` retrieves the list of metrics and dimensions that a RUM app monitor is sending to a single destination. ::
+
+	aws rum batch-get-rum-metric-definitions \
+		--app-monitor-name MyWebApp \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general